<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tags</class>
 <widget class="QWidget" name="tags">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
   <item row="4" column="1">
    <widget class="QCheckBox" name="check_medical">
     <property name="text">
      <string>MEDICAL</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="check_residential">
     <property name="text">
      <string>RESIDENTIAL</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="check_food">
     <property name="text">
      <string>FOOD</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="check_police">
     <property name="text">
      <string>POLICE</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="check_millitary">
     <property name="text">
      <string>MILITARY</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="check_common">
     <property name="text">
      <string>COMMON</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="check_fuel">
     <property name="text">
      <string>FUEL</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="check_guns">
     <property name="text">
      <string>GUNS</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
